draft-ietf-ccwg-bbr.md

```diff
@@ -2146,26 +2146,22 @@ rate at which the network path can deliver data, and not simply the rate
 at which the application happens to be transmitting data at any moment.
 
 To track this, the estimator marks a bandwidth sample as application-limited
-if there was some moment during the sampled flight of data packets when there
-was no data ready to send.
-
-The algorithm detects that an application-limited phase has started when
-the sending application requests to send new data, or the connection's
-retransmission mechanisms decide to retransmit data, and the connection meets
-all of the following conditions:
-
-1. The transport send buffer has less than one C.SMSS of unsent data available
-  to send.
-
-2. The sending flow is not currently in the process of transmitting a packet.
-
-3. The amount of data considered in flight is less than the congestion window
-  (C.cwnd).
-
-4. All the packets considered lost have been retransmitted.
-
-
-If these conditions are all met then the sender has run out of data to feed the
+if there was some moment during the sampled flight of data packets when
+congestion control would have allowed a send, and yet there was no data ready
+to send.
+
+More specifically, the algorithm detects that an application-limited phase has
+started when the sending application requests to send new data,
+or the connection's retransmission mechanisms decide to retransmit data,
+and the connection meets the following conditions: the congestion window and
+pacing rate would have allowed the connection to send data, and yet the
+connection is not currently sending data and has no data to send (i.e., no
+unsent/new data, no retransmit data, and no other data, such as encoded data
+blocks for forward error correction). The precise determination of this
```

Comment:
There are a lot of reasons one might want to send a packet that counts towards inflight, I'd prefer to focus on the two most common.

```suggestion
unsent data or retransmissions of previously sent data). The precise determination of this
```